content/docs/develop/usecases/onboard-carts-service/index.md

```diff
@@ -35,21 +35,25 @@ To illustrate the scenario this use case addresses, keptn relies on two services
 1. Git clone artifacts for this use case.
 
     ```console
-    cd ~
-    git clone https://github.com/keptn/examples.git
-    cd ~/examples/onboarding-carts
+    $ cd ~
```

Comment:
please get rid of the $ signs in the command - also maybe chain the commands using && so that a user can copy and execute the command without needing to adapt it